src/test-runner/assertions.ts

```diff
@@ -0,0 +1,382 @@
+const chai = require("chai");
+import { ApiPromise, Keyring } from "@polkadot/api";
+const utilCrypto = require("@polkadot/util-crypto");
+import minimatch from "minimatch";
+import path from "path";
+import { BackchannelMap } from ".";
+import { DEFAULT_INDIVIDUAL_TEST_TIMEOUT } from "../constants";
+import {
+  chainCustomSectionUpgrade,
+  chainUpgradeFromLocalFile,
+  chainUpgradeFromUrl,
+  connect,
+  findPatternInSystemEventSubscription,
+  validateRuntimeCode,
+} from "../jsapi-helpers";
+import { Network } from "../network";
+import { FnArgs } from "../types";
+const { assert, expect } = chai;
+import { JSDOM } from "jsdom";
+import { decorators } from "../utils/colors";
+import { isValidHttpUrl } from "../utils/misc";
+
+// helper
+function toChaiComparator(op: string): string {
+  return op.charAt(0).toLocaleLowerCase() + op.slice(1);
+}
+
+const IsUp = (args: FnArgs) => {
+  const { node_name, timeout } = args;
```

Comment:
```suggestion
const IsUp = ({ node_name, timeout }: FnArgs) => {
```

Reply:
Nice! I like this shortcuts :) . I will change those in one commit :)